assets/js/blocks/cart-checkout/cart/style.scss

```diff
@@ -238,13 +238,6 @@ table.wc-block-cart-items {
 		}
 	}
 
-	.wc-block-cart__sidebar {
-		& > div:not(.wc-block-components-totals-wrapper) {
```

Comment:
I can see `.wc-block-components-totals-wrapper` in both checkout and cart. This solution isn't bullet proof though because at a certain width, the browser will decide to wrap the input element and the button. The ideal solution would be to set `flex-wrap: no-wrap` on the `.wc-points-rewards-redeem-form` element but that needs to be done in the points and rewards app.